src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,20 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private final StorageDao storageDao;
+    private final UserValidator userValidator;
+
+    public RegistrationServiceImpl() {
+        storageDao = new StorageDaoImpl();
+        userValidator = new UserValidator();
+    }
 
     @Override
     public User register(User user) {
-        return null;
+        userValidator.validateAge(user);
+        userValidator.validateLogin(user);
+        userValidator.validatePassword(user);
```

Comment:
combine validation methods into one. 
```suggestion
        userValidator.validateAge(user);
        userValidator.validateLogin(user);
        userValidator.validatePassword(user);
```